tracer/errors.go

```diff
@@ -143,10 +59,9 @@ func aggregateErrors(errChan <-chan error) map[string]errorSummary {
 // logErrors logs the errors, preventing log file flooding, when there
 // are many messages, it caps them and shows a quick summary.
 // As of today it only logs using standard golang log package, but
-// later we could send those stats to agent [TODO:christian].
+// later we could send those stats to agent // TODO(ufoot).
```

Comment:
I have a task in my backlog to create a `log` package for the tracer. This package would allow setting verbosity levels (info, debug, silent, etc) for globally for the entire package. It will also take an external logger as an interface. That external logger could be forwarding messages to the agent like you say.